contracts/v6/OraclePrice.sol

```diff
@@ -69,7 +55,10 @@ contract OraclePrice is Ownable, WhitelistAdminRole, KeeperCompatibleInterface,
         performData = checkData;
     }
 
-    function performUpkeep(bytes calldata performData) external override onlyWhitelistAdmin {
```

Comment:
Why have we removed the guard to be only be callable by a whitelisted admin?

Reply:
I was added this address but we don't know keepers address who will call this method so i removed this guard.
Keepers may call this method from any address and if address is not whiteliste then request will fail so i removed this guard.